### ISSUE TITLE
[Bug]: ReactiveUI.Blazor - Using ReactiveInjectableComponentBase<T> renders the view twice

### ISSUE THREAD

**Issue by Lukzy** [reactiveui/ReactiveUI#3256]
### Describe the bug 🐞

If I inherit from `ReactiveInjectableComponentBase<T>` to attach a ViewModel with a View, the view gets rendered twice on each activation.

For comparison, the same View and ViewModel with the ViewModel beeing attached by an `[Inject]` attribute, the View is rendered only once.

### Step to reproduce

Check the linked reproduction repository.

- `ViewWithInherits.razor` uses `@inherits ReactiveInjectableComponentBase<T>` and will be rendered twice
- `ViewWithInject.razor` uses `[Inject]` attribute and will be rendered once

I verified the number of render cycles with the debugger, the counter property and Trace message (not in the example). 

### Reproduction repository

https://github.com/Lukzy/ReactiveUI.ExecutedTwiceBug

### Expected behavior

Both ways of attaching a ViewModel should behave the same. I would expect the View to be only rendered once.

### IDE

Visual Studio 2022

### Operating system

Windows 10 Enterprise

### Version

10.0.19044 Build 19044

### Device

LENOVO X1

### ReactiveUI Version

18.0.10

### Additional information ℹ️

Behavior tested with these browsers:
- Microsoft Edge - Version 101.0.1210.32 (Official build) (64-bit)
- Google Chrome - Version 100.0.4896.127 (Official Build) (64-bit)
- Firefox - Version 97.0.2 (32-bit)

**Comment by iashaik** [comment]
I spent few hours trying to figure out what I did that was wrong.

**Comment by glennawatson** [comment]
Closing this issue as part of a cleanup of long-standing issues. The double-render behavior in ReactiveInjectableComponentBase stems from the OnAfterRender lifecycle calling StateHasChanged. The Blazor integration has been updated since this was reported. If you're still seeing this on a current version, please open a new issue with a reproduction. Thank you for reporting it.
